packages/tokens/README.md

```diff
@@ -135,14 +135,17 @@ console.log(div.style.backgroundColor);
 
 Our variables are also available as scss variables that make it possible to consume them in sass based projects.
 
+> Note that the SASS compiler does not ship with built in support for `node_modules`. You will either have to use relative paths or use a loader like [node-sass-package-importer](https://www.npmjs.com/package/node-sass-package-importer) or [sass-module-importer](https://www.npmjs.com/package/sass-module-importer).
+> Configuration for this is NOT part of this guide.
+
 ```scss
 // Import the design tokens first.
 // This can be done in a sass file or in any other way described above.
 // Make sure to NOT add the .css file suffix, this will not work in sass
-@import "~@synergy-design-system/tokens/themes/light";
+@import "./node_modules/@synergy-design-system/tokens/themes/light";
```

Comment:
I tested it in a vite project. 
![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/4ba132c5-3420-4cb0-bec9-7e3f969de06f)

So it seems on relative imports we need the **dist** in the import path, otherwise not.
Therefore we should update the import here to also have the **dist** in the path

Reply:
Yes, I think this is caused by the fact vite and its plugins honor the `exports` statement in `package.json`, but other bundlers ignore it (like webpack). On relative paths it ignores `package.json` completely, so maybe it is more save to use the `dist` folder always for scss before messing different paths up for each separate bundler. What do you think?

Reply:
Resolved in latest commit